src/deluge/model/global_effectable/global_effectable.cpp

```diff
@@ -156,11 +168,23 @@ void GlobalEffectable::displayCompressorAndReverbSettings(bool on) {
 	}
 	else {
 		if (on) {
-			display->displayPopup(getCompressorModeDisplayName());
+			if (getSelectedClip()) {
+				// We are inside a clip (either audioclip or affect-entire kit): show Sidechain sync
+				if (compressor.syncLevel == SYNC_LEVEL_32ND) {
```

Comment:
is this using the same logic as Sound::displaySidechainAndReverbSettings and Sound::getSidechainDisplayName?

Reply:
Yes, it should be